vnext/Desktop/React.Windows.Desktop.vcxproj.filters

```diff
@@ -81,6 +81,7 @@
     <Midl Include="ABI\NativeTracing.idl">
       <Filter>ABI</Filter>
     </Midl>
+    <Midl Include="..\Microsoft.ReactNative\ReactInstance.idl" />
```

Comment:
><Midl Include="..\Microsoft.ReactNative\ReactInstance.idl" /> [](start = 4, length = 61)

I wonder why is it needed. I want to remove this file soon.

Reply:
Never mind - it is probably added automatically by VS.

---
In reply to: [428989823](https://github.com/microsoft/react-native-windows/pull/4984#discussion_r428989823) [](ancestors = 428989823)

Reply:
It's not needed, but Visual Studio added it automatically, as it was part of the referenced files in the VCXPROJ.

Note, this file is just the .FILTERS, for IDE displaying purposes.
If you remove the file from the project, it will also disappear from the filter.

This change is innocuous.